src/System.Data.SqlClient/src/project.json

```diff
@@ -13,7 +13,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.Net.Primitives": "4.0.10",
     "System.Net.Security": "4.0.0-beta-*",
-    "System.Net.Sockets": "4.0.10-beta-*",
+    "System.Net.Sockets": "4.1.0-beta-*",
```

Comment:
What's your plan to manage the change distribution? Create a new version for Net.Sockets? 
What does it mean for us as the consumers? 
We need to have Data.SqlClient run on the Networking for XPlat for which the version is 4.1.0-beta-*